pkg/core/extension_store.go

```diff
@@ -503,6 +528,8 @@ func (store *ExtensionStore) UpdateService(ctx context.Context, service *domain.
 	if err != nil {
 		return err
 	}
+	service.Registered = svcRecord.Registered
```

Comment:
Why?

Reply:
Same as before: the registered timestamp is immutable for the lifetime of an extension/service/set of credentials.